packages/@aws-cdk/custom-resource-handlers/lib/custom-resources/aws-custom-resource-handler/index.ts

```diff
@@ -1,17 +1,277 @@
 /* eslint-disable @typescript-eslint/no-require-imports */
-export { PHYSICAL_RESOURCE_ID_REFERENCE } from './shared';
+/* eslint-disable no-console */
+import { execSync } from 'child_process';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
We need this because the adapter is in the `package.json` as a `devDependency` and it is bundled with the handler. This eslint exception is there so that eslint doesn't complain about the adapter not being a `dependency`.